app/Makefile

```diff
@@ -128,126 +88,24 @@ ifndef ICONNAME
 $(error ICONNAME is not set)
 endif
 
-############
-# Platform
-
-DEFINES       += PRINTF\(...\)=
-
-APPVERSION=$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)
-DEFINES       += APPVERSION=\"$(APPVERSION)\"
-
-DEFINES       += OS_IO_SEPROXYHAL
-DEFINES       += HAVE_BAGL HAVE_SPRINTF
-DEFINES       += HAVE_IO_USB HAVE_L4_USBLIB IO_USB_MAX_ENDPOINTS=7 IO_HID_EP_LENGTH=64 HAVE_USB_APDU
-
-DEFINES       += LEDGER_MAJOR_VERSION=$(APPVERSION_M) LEDGER_MINOR_VERSION=$(APPVERSION_N) LEDGER_PATCH_VERSION=$(APPVERSION_P)
-
-DEFINES       += USB_SEGMENT_SIZE=64
-DEFINES       += HAVE_BOLOS_APP_STACK_CANARY
-
-DEFINES       += HAVE_WEBUSB WEBUSB_URL_SIZE_B=0 WEBUSB_URL=""
-
-ifeq ($(TARGET_NAME),TARGET_NANOS)
-DEFINES       += IO_SEPROXYHAL_BUFFER_SIZE_B=128
-else
-# Assume Nano X/S+
-DEFINES       += IO_SEPROXYHAL_BUFFER_SIZE_B=300
-
-DEFINES       += HAVE_GLO096
-DEFINES       += HAVE_BAGL BAGL_WIDTH=128 BAGL_HEIGHT=64
-DEFINES       += HAVE_BAGL_ELLIPSIS # long label truncation feature
-DEFINES       += HAVE_BAGL_FONT_OPEN_SANS_REGULAR_11PX
-DEFINES       += HAVE_BAGL_FONT_OPEN_SANS_EXTRABOLD_11PX
-DEFINES       += HAVE_BAGL_FONT_OPEN_SANS_LIGHT_16PX
-
-DEFINES       += HAVE_UX_FLOW
-endif
-
-ifeq ($(TARGET_NAME),TARGET_NANOX)
-# X specific
-DEFINES       += HAVE_BLE
-DEFINES       += HAVE_BLE_APDU BLE_COMMAND_TIMEOUT_MS=2000
+include $(CURDIR)/../deps/ledger-zxlib/makefiles/Makefile.platform
+LDFLAGS += -z muldefs
 
-SDK_SOURCE_PATH  += lib_blewbxx lib_blewbxx_impl
-endif
-
-# App specific
-
-#Feature temporarily disabled
-DEFINES   += LEDGER_SPECIFIC
-
-# Add security review banner. To be removed once Ledger security review is done.
-APP_LOAD_PARAMS += --tlvraw 9F:01
-DEFINES += HAVE_PENDING_REVIEW_SCREEN
```

Comment:
security review banner removed as requested in [B2CA-1230](https://ledgerhq.atlassian.net/browse/B2CA-1230)